src/test/java/utilities/UtilityReader.java

```diff
@@ -0,0 +1,13 @@
+package utilities;
+
+import java.util.List;
+
+/**
+ * This reader is needed only to test results of WriterImplTest
+ * and FruitServiceImplTest
+ */
```

Comment:
it's better to rely on Java's built-in mechanisms for that, Utility may itself need testing